tests/views/definitions/unnestEmptyCollection.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Unnest empty collection",
+  "resource": "Patient",
+  "select": [
+    {
+      "name": "id",
+      "expr": "id"
+    },
+    {
+      "forEach": "photo",
```

Comment:
The example output for this is emitting an empty column for photo_title since there are no values, but I think it would emit no rows whatsoever. The output seen here is what I'd expect for a proposed `forEachOrNull` operator (that we still need to add in).

I'm not sure which meeting it was, but we had discussed that `forEach` would emit nothing if there is no value, but we would add a `forEachOrNull` that works as seen in the example. This way someone unnesting an address table from Patient wouldn't see rows with empty addresses in the case of patients who didn't have any.

Reply:
Fixed in #84